crates/codegen/parser/src/rust_lib_code_generator.rs

```diff
@@ -25,31 +25,23 @@ impl CodeGenerator {
             )
             .unwrap();
 
-        {
-            let cst_visitor = codegen
-                .read_file(
-                    &codegen
-                        .repo_root
-                        .join("crates/codegen/parser_templates/src/rust/cst_visitor.rs"),
-                )
-                .unwrap();
-            codegen
-                .write_file(&output_dir.join("cst_visitor.rs"), &cst_visitor)
-                .unwrap();
-        }
+        codegen
```

Comment:
Should we rename `RuleKind` to `ParserKind` and `TokenKind` to `ScannerKind` to match the new schema?

Reply:
Later issue as well.

Reply:
* https://github.com/NomicFoundation/slang/issues/373